TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -0,0 +1,21 @@
+@objc public class TradeItSDK: NSObject {
+    private static var apiKey: String?
+    private static var environment: TradeitEmsEnvironments?
+
+    public static let launcher = TradeItLauncher()
+    private (set) public static var linkedBrokerManager: TradeItLinkedBrokerManager!
+    private (set) public static var marketDataService: TradeItMarketService!
+    private (set) public static var brokerCenterService: TradeItBrokerCenterService!
```

Comment:
Hmm, interesting. I can't think of a better way of getting the desired effect. Let's do it.